tests/ciscotest.rb

```diff
@@ -221,4 +223,22 @@ def mt_full_interface?
 
     "ethernet#{slot}/1"
   end
+
+  def remove_all_vlans
```

Comment:
Nit: please move your new remove_all_* methods up to where the other remove_all_* methods live and keep them in alpha order. We will likely have a few more of these remove_all's coming...